### PR TITLE
UI fixes on About Page

### PR DESCRIPTION
**Pull request by vazquez**

### CODE REVIEW

**Comment by vazquez** [comment]
For #337

File: pages/about.jsx [hunk 1]
```diff
@@ -63,13 +58,14 @@ var OurFocus = React.createClass({
...
-          </p>
-          <img src="/img/shared-iconography/icon-horizontal-divider.svg" alt=""/>
+        <section className="row page-end our-goal">
+          <ImageTag alt="A stylised icon of a city"
+              className="center-block" src1x="/img/about-page/icon-city.jpg"
+              src2x="/img/about-page/icon-city@2x.jpg" width={292} height={102} />
+            <p className="learn-more">
+              In 2015, our goal is to foster and sustain web literacy activities in 500 cities around the globe. Join us by teaching, learning, sharing and imagining the full power of the web.
+            </p>
+            <img src="/img/shared-iconography/icon-horizontal-divider.svg" alt="" className="center-block"/>
```
Comment (alicoding) [inline pages/about.jsx]: hmm why leave `alt` empty?

**Comment by alicoding** [comment]
r+ with one question to address.

File: pages/about.jsx (hunk 1 above)
Comment (vazquez) [inline pages/about.jsx]: Because its purely decoration.

Comment (alicoding) [inline pages/about.jsx]: http://www.w3.org/WAI/tutorials/images/decorative/ :+1:

**Pull request merged**